src/gui/Application.cpp

```diff
@@ -326,7 +341,17 @@ bool Application::isAlreadyRunning() const
     return config()->get(Config::SingleInstance).toBool() && m_alreadyRunning;
 }
 
-bool Application::sendFileNamesToRunningInstance(const QStringList& fileNames)
+/**
+ * Send data to the running UI instance; currently used to lock databases via the CLI or send new files to open.
+ *
+ * Right now nullptr fileNames is sent as long as it is not empty. This is not sustainable; the sent data (if any)
```

Comment:
this comment is not true, you are not sending nullptr you are sending an empty list.